experiment/simulation/index.html

```diff
@@ -1,13 +1,80 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-	<!-- Add CSS at the head of HTML file -->
-    <link rel="stylesheet" href="./css/main.css">
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css">
```

Comment:
Remove Bulma import if you are using virtual-style. If something breaks, please talk to Shlok to include that in the virtual-style library

Reply:
yes sir some of the components broke, I would contact shlok regarding this.